src/serde/deserialize_program.rs

```diff
@@ -1267,6 +1279,7 @@ mod tests {
 
         let program_json: ProgramJson = serde_json::from_str(valid_json).unwrap();
 
+        #[cfg(feature = "std")]
```

Comment:
`InputFile { filename: String::from("test/contracts/cairo/always_fail.cairo") },`
this should be behind the flag. not the whole debug_info